chat-widget/src/components/livechatwidget/common/endChat.ts

```diff
@@ -85,7 +86,7 @@ const prepareEndChat = async (props: ILiveChatWidgetProps, chatSDK: any, state:
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const endChat = async (props: ILiveChatWidgetProps, chatSDK: any, state: ILiveChatWidgetContext, dispatch: Dispatch<ILiveChatWidgetAction>, setAdapter: any, setWebChatStyles: any, adapter: any,
-    skipEndChatSDK?: boolean, skipCloseChat?: boolean, postMessageToOtherTab?: boolean, uwid = "") => {
+    skipEndChatSDK?: boolean, skipCloseChat?: boolean, postMessageToOtherTab?: boolean) => {
     if (!skipEndChatSDK && chatSDK.conversation) {
         try {
```

Comment:
Do we have unit test against these changes ?

Reply:
We do not yet, as the global state makes unit testing harder. We have the work planned in backlog [User Story 3658462](https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3658462): Engineering Improvement - Unit Tests